### PR TITLE
economia.ig.com.br

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/117705

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-06 в 16 02 16

Снимок экрана 2022-05-06 в 16 02 26

</details><br/>

### CODE REVIEW

File: SpanishFilter/sections/specific.txt [hunk 1]
```diff
@@ -19,6 +19,8 @@ atribuna.com.br,jovemnerd.com.br,laopiniondemurcia.es,larazon.es,dicio.com.br,ba
 brasilescola.uol.com.br##.publi-dir
 brasilescola.uol.com.br##.corpoPubli
 brasilescola.uol.com.br##.area-publi-topo
+economia.ig.com.br##div[class^="maislidas"]
+economia.ig.com.br###containerComp
```
Comment (Alex-302) [inline SpanishFilter/sections/specific.txt:23]: It seems this is not related to ads. I see these elements at https://ultimosegundo.ig.com.br/policia/2022-05-06/ataque-faca-alunos-feridos-escola-ilha-governador-rj.html and useful content may be blocked.

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]
